app/dashboard/views.py

```diff
@@ -1788,31 +1788,61 @@ def profile_details(request, handle):
     except (ProfileNotFoundException, ProfileHiddenException):
         raise Http404
 
-    bounties = profile.bounties
-    for word in profile.keywords:
-        bounties = bounties.keyword(word) # TODO: Is this right
+    if not settings.DEBUG:
+        network = 'mainnet'
+    else:
+        network = 'rinkeby'
+
+    keywords = request.GET.get('keywords')
+
+    # bounties = profile.bounties
+        # Prefetch('feedbacks', queryset=queryset, to_attr='feedback'), 'feedback__rating') \
+    # queryset = FeedbackEntry.objects.all().filter(receiver_profile=profile)
+    bounties = Bounty.objects.current().prefetch_related('fulfillments', 'interested', 'interested__profile', 'feedbacks') \
+        .filter(interested__profile=profile, network=network,) \
+        .filter(interested__status='okay') \
+        .filter(interested__pending=False).filter(idx_status='done') \
+        .filter(feedbacks__receiver_profile=profile) \
+        .filter(
+            Q(metadata__issueKeywords__icontains=keywords) | \
+            Q(title__icontains=keywords) | \
+            Q(issue_description__icontains=keywords)
+        ) \
+        .distinct('pk')
+
+
+
+    # for word in keywords:
+        # bounties = bounties.keyword(word) # TODO: Is this right
 
     _bounties = []
+    _orgs = []
     if bounties :
         for bounty in bounties:
+
             _bounty = {
                 'title': bounty.title,
-                'url': bounty.get_absolute_url()
+                'url': bounty.get_absolute_url(),
+                'id': bounty.id,
+                'org': bounty.org_name,
+                'rating': [feedback.rating for feedback in bounty.feedbacks.all().distinct('bounty_id')],
```

Comment:
This would show both funder + contributor rating but is more efficient! 🤔 
would it make sense to add `.filter(receiver_profile__isnull=False)` ?
Also does it need to be an array  ?